src/shared/defaultAwsContextCommands.ts

```diff
@@ -280,18 +282,19 @@ export class DefaultAWSContextCommands {
      * regions are shown. Regions provided must exist in the available regions to be shown.
      */
     private async promptForRegion(regions?: string[], title?: string): Promise<string | undefined> {
```

Comment:
nit: `regions` would be better as `regionsToShow`, on first glance I didn't get the point of what they were, then the bottom one could be `filteredRegions` or something (the doc comment was above the diff area)